<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="trem_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>79</red>
          <green>129</green>
          <blue>189</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>79</red>
          <green>129</green>
          <blue>189</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>79</red>
          <green>129</green>
          <blue>189</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>79</red>
          <green>129</green>
          <blue>189</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>79</red>
          <green>129</green>
          <blue>189</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>79</red>
          <green>129</green>
          <blue>189</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>79</red>
          <green>129</green>
          <blue>189</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>79</red>
          <green>129</green>
          <blue>189</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>79</red>
          <green>129</green>
          <blue>189</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background-color: rgb(79, 129, 189);border: 2px solid rgb(64, 104, 152)}</string>
    </property>
    <property name="text">
     <string>T1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>120</width>
      <height>5</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {background-color: rgb(84, 132, 191);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>5</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {background-color: rgb(84, 132, 191);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="trem_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {background-color: rgb(192, 80, 77);border: 2px solid rgb(154, 64, 62)}</string>
    </property>
    <property name="text">
     <string>T2</string>
    </property>
    <property name="indent">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>120</width>
      <height>5</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {background-color: rgb(84, 132, 191);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>120</width>
      <height>5</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {background-color: rgb(84, 132, 191);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>120</width>
      <height>5</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {background-color: rgb(84, 132, 191);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>5</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {background-color: rgb(84, 132, 191);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>5</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {background-color: rgb(84, 132, 191);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_6">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>120</width>
      <height>5</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {background-color: rgb(84, 132, 191);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_7">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>140</y>
      <width>125</width>
      <height>5</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {background-color: rgb(84, 132, 191);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4_4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>5</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {background-color: rgb(84, 132, 191);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4_5">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>140</y>
      <width>5</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {background-color: rgb(84, 132, 191);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_8">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>260</y>
      <width>125</width>
      <height>5</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {background-color: rgb(84, 132, 191);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_9">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>260</y>
      <width>120</width>
      <height>5</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {background-color: rgb(84, 132, 191);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4_6">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>140</y>
      <width>5</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {background-color: rgb(84, 132, 191);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4_7">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>5</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {background-color: rgb(84, 132, 191);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="trem_1_slider">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>40</y>
      <width>160</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::handle:horizontal {
    background-color: rgb(79, 129, 189);
    border: 1px solid rgb(56, 93, 138) ;
    width: 40px;
    height: 40px;
}</string>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickInterval">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSlider" name="trem_5_slider">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>240</y>
      <width>160</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::handle:horizontal {
    background-color: rgb(247, 150, 70);
    border: 1px solid rgb(182, 109, 49);
    height: 40px;
    width: 40px;
}
</string>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickInterval">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSlider" name="trem_2_slider">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>90</y>
      <width>160</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::handle:horizontal {
    background-color: rgb(192, 80, 77);
    border: 1px solid rgb(140, 56, 54);
    height: 40px;
    width: 40px;
}
</string>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickInterval">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSlider" name="trem_4_slider">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>190</y>
      <width>160</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::handle:horizontal {
    background-color:rgb(128, 100, 162);
    border: 1px solid rgb(92, 71, 118);
    height: 40px;
    width: 40px;
}
</string>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickInterval">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSlider" name="trem_3_slider">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>140</y>
      <width>160</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::handle:horizontal {
    background-color: rgb(155, 187, 89);
    border: 1px solid rgb(113, 137, 63) ;
    height: 40px;
    width: 40px;
}
</string>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickInterval">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="trem_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>70</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background-color: rgb(155, 187, 89);border: 2px solid rgb(113, 137, 63)}</string>
    </property>
    <property name="text">
     <string>T3</string>
    </property>
    <property name="indent">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLabel" name="trem_4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>250</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background-color: rgb(128, 100, 162);border: 2px solid rgb(92, 71, 118)}</string>
    </property>
    <property name="text">
     <string>T4</string>
    </property>
    <property name="indent">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLabel" name="trem_5">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>250</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background-color: rgb(247, 150, 70);border: 2px solid rgb(182, 109, 49)}</string>
    </property>
    <property name="text">
     <string>T5</string>
    </property>
    <property name="indent">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>40</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Trem 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>190</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Trem 4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>140</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Trem 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>90</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Trem 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>240</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Trem 5</string>
    </property>
   </widget>
   <widget class="QLabel" name="velocidade_trem_1">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>40</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="velocidade_trem_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>90</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="velocidade_trem_3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>140</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="velocidade_trem_4">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>190</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="velocidade_trem_5">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>240</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <zorder>label_trilho1</zorder>
   <zorder>label_trilho4</zorder>
   <zorder>label_trilho1_2</zorder>
   <zorder>label_trilho1_3</zorder>
   <zorder>label_trilho1_4</zorder>
   <zorder>label_trilho4_2</zorder>
   <zorder>label_trilho4_3</zorder>
   <zorder>label_trilho1_6</zorder>
   <zorder>label_trilho1_7</zorder>
   <zorder>label_trilho4_4</zorder>
   <zorder>label_trilho4_5</zorder>
   <zorder>label_trilho1_8</zorder>
   <zorder>label_trilho1_9</zorder>
   <zorder>label_trilho4_6</zorder>
   <zorder>label_trilho4_7</zorder>
   <zorder>trem_1_slider</zorder>
   <zorder>trem_5_slider</zorder>
   <zorder>trem_2_slider</zorder>
   <zorder>trem_4_slider</zorder>
   <zorder>trem_3_slider</zorder>
   <zorder>trem_1</zorder>
   <zorder>trem_2</zorder>
   <zorder>trem_3</zorder>
   <zorder>trem_4</zorder>
   <zorder>trem_5</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_6</zorder>
   <zorder>velocidade_trem_1</zorder>
   <zorder>velocidade_trem_2</zorder>
   <zorder>velocidade_trem_3</zorder>
   <zorder>velocidade_trem_4</zorder>
   <zorder>velocidade_trem_5</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>651</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
